src/components/UserCard/UserCard.styles.ts

```diff
@@ -74,3 +86,76 @@ export const ReportBtn = styled.div`
   grid-area: ReportBtn;
   margin-top: ${theme.spacing(2)}px;
 `
+
+export const PartnerContainer = styled(Typography).attrs(() => ({
+  variant: variants.bodyRegular,
+  component: 'div',
+}))`
+  display: grid;
+  @media ${devices.desktop} {
+    grid-template-areas:
+        ". Avatar ."
+        ". Name ."
+        "Description Description Description"
+        "DonationsNeeds DonationsNeeds DonationsNeeds"
+        "VolunteersNeeds VolunteersNeeds VolunteersNeeds"
+        "Phone Mail Website";
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+  @media ${devices.mobile} {
+    grid-template-areas:
+      "Avatar"
+      "Name"
+      "Description"
+      "DonationsNeeds"
+      "VolunteersNeeds"
+      "Phone"
+      "Mail"
+      "Website";
+    grid-template-columns: 1fr;
+  }
+  max-width: 100%;
+  margin-bottom: ${theme.spacing(2)}px;
+`
+
+export const PhoneArea = styled.div`
+  grid-area: Phone;
+  @media ${devices.desktop} {
+    display: grid;
+  }
+  @media ${devices.mobile} {
+    display: flex;
+  }
+`
+
+export const WebsiteArea = styled.div`
+  grid-area: Website;
```

Comment:
J'ai tendance à nommer tous les `grid-area` avec le même nom que le composant. D'ailleurs ici le terme `Area` n'est pas utile je trouve

```suggestion
export const Website = styled.div`
  grid-area: Website;
```

Reply:
à modifier pour les autres composant et `grid-areat`